src/blocks/button.css

```diff
@@ -0,0 +1,22 @@
+.card__button {
+  display: flex;
+}
```

Comment:
This styles should be in card.css file
Button - it's not the separate BEM block